build.gradle

```diff
@@ -117,6 +116,33 @@ if (new File('.git').exists()) {
 
 subprojects {
   apply plugin: 'java'
+  if (!project.name.equals('connect')) {
+    apply plugin: 'eclipse'
+
+    eclipse.classpath.file {
```

Comment:
It's a good idea to add a comment explaining the reasoning for this. Also, maybe it should be in a separate method.